web/src/engine/websites/MangaGecko.ts

```diff
@@ -0,0 +1,47 @@
+import { Tags } from '../Tags';
+import icon from './MangaGecko.webp';
+import { Chapter, DecoratableMangaScraper, type MangaPlugin, type Manga } from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+import { FetchCSS, FetchRequest } from '../FetchProvider';
+
+@Common.MangaCSS(/^{origin}\/manga\/[^/]+\/$/, 'div.main-head h1[itemprop="name"]')
+@Common.PagesSinglePageCSS('section.page-in div img[onerror]')
+@Common.ImageAjax()
+export default class extends DecoratableMangaScraper {
+
+    public constructor() {
+        super('mangagecko', `MangaGecko`, 'https://www.mgeko.com', Tags.Media.Manga, Tags.Media.Manhua, Tags.Media.Manhwa, Tags.Language.English, Tags.Source.Aggregator);
+    }
+
+    public override get Icon() {
+        return icon;
+    }
+
+    public override async FetchMangas(provider: MangaPlugin): Promise<Manga[]> {
+
+        //get pagination because it never ends otherwise
```

Comment:
Comment is redundant, source code already tells the developer what is happening

Reply:
You may have asked why i did it like that, but ok :p